codebuild/test_vectors/nodejs_latest.yml

```diff
@@ -0,0 +1,27 @@
+version: 0.2
```

Comment:
This is the wrong name. :)
This should be publishing tests or something.
Since all this does is test the publishing process.